modules/ipfs-cpinner-provider/src/IpfsClient.ts

```diff
@@ -9,9 +8,15 @@ export default class implements IpfsClient {
   }
 
   async get (cid: string): Promise<string> {
-    const buffer = await all(this.ipfsHttpClient.cat(cid))
+    const responses = await this.ipfsHttpClient.cat(cid)
+    const decoder = new TextDecoder()
+    let data = ''
 
-    return buffer.toString()
+    for await (const chunk of responses) {
```

Comment:
I think `await` is not necessary here, as the `responses` variable is already `await`ed in line 11

Reply:
Hey, thanks for pointing that out, I think I can safely remove the first await on line 11 since it only returns the generator function and keep the awaits for the following `yields`